api/models/process.go

```diff
@@ -294,9 +296,9 @@ func ListPendingProcesses(app string) ([]*Process, error) {
 			tres, err := ECS().DescribeTaskDefinition(&ecs.DescribeTaskDefinitionInput{
 				TaskDefinition: d.TaskDefinition,
 			})
-
 			if err != nil {
-				return nil, err
+				fmt.Printf("ns=kernel at=ListPendingProcesses state=error message=\"%s\"\n", err)
```

Comment:
I see that line 282 can return an error. I think this particular function should pick one strategy or another in both places and either log and not error or error. If I had to make the call I'd say it can still error in both places.